noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -41,8 +41,17 @@ impl NullifiableNote for AddressNote {
         )
     }
 
-    unconstrained fn compute_nullifier_without_context(self, storage_slot: Field) -> Field {
-        let note_hash_for_nullify = compute_note_hash_for_nullify(self, storage_slot);
+    unconstrained fn compute_nullifier_without_context(
+        self,
+        storage_slot: Field,
+        contract_address: AztecAddress,
+        note_nonce: Field,
```

Comment:
I was considering making this a static function accepting just `RetrievedNote<Self>` but `note_hash_counter` was not needed to be passed here so I decided against it.

Would consider refactoring this in https://github.com/AztecProtocol/aztec-packages/issues/8589

Reply:
Yeah, the utils hash fns need to be changed to not have to deal with this annoyance. But that can wait.